analytics_data_api/v0/views/courses.py

```diff
@@ -689,6 +689,54 @@ def get_queryset(self):
         return rows
 
 
+# pylint: disable=abstract-method
+class ProblemsAndTagsListView(BaseCourseView):
+    """
+    Get the problems with the connected tags.
+
+    **Example request**
+
+        GET /api/v0/courses/{course_id}/problems_and_tags/
+
+    **Response Values**
+
+        Returns a collection of submission counts and tags for each problem. Each collection contains:
+
+            * module_id: The ID of the problem.
+            * tag_name: Name of the connected tag
+            * tag_value: Value of the connected tag
```

Comment:
I think this description isn't quite right.  It looks like it's more like "tags" which is a map of tag names to values.